examples/gallery/3d_plots/scatter3d.py

```diff
@@ -20,6 +20,10 @@
 # Load sample iris data and convert 'species' column to categorical dtype
 df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/iris.csv")
 df.species = df.species.astype(dtype="category")
```

Comment:
> `df.species = df.species.astype(dtype="category")`

The "species" column is converted to categorical type by this code. I think we can add a comment noting that they are in alphabetical order by default. If anyone wants to adjust the order, they should look at https://pandas.pydata.org/docs/user_guide/categorical.html instead.